lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js.bc.32b.h

```diff
@@ -135,7 +135,7 @@ namespace Js
 {
     const char Library_Bytecode_JsBuiltIn[] = {
 /* 00000000 */ 0x43, 0x68, 0x42, 0x63, 0x0E, 0x13, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
-/* 00000010 */ 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0xFE, 0xE8, 0x02, 0x00, 0xFE,
+/* 00000010 */ 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x20, 0x00, 0xFE, 0xF0, 0x02, 0x00, 0xFE,
```

Comment:
>0xF0 [](start = 87, length = 4)

change to this bytecode with no source change indicates a new GUID is needed

Reply:
Sure, technically, but at this point the guid has changed like 3 times in this branch without ever having been merged back to master. I don't think theres any need.